src/main.c

```diff
@@ -1540,7 +1541,8 @@ static gchar* r_status_formatter_readable(RaucStatusPrint *status)
 	g_string_append(text, "=== System Info ===\n");
 	g_string_append_printf(text, "Compatible:  %s\n", status->compatible);
 	g_string_append_printf(text, "Variant:     %s\n", status->variant);
-	g_string_append_printf(text, "Booted from: %s (%s)\n\n", bootedfrom ? bootedfrom->name : NULL, status->bootslot);
+	g_string_append_printf(text, "Booted from: %s (%s)\n", bootedfrom ? bootedfrom->name : NULL, status->bootslot);
+	g_string_append_printf(text, "Counter lock: %s\n\n", status->lock_counter ? "enabled" : "disabled");
```

Comment:
```suggestion
//	g_string_append_printf(text, "Counter lock: %s\n\n", status->lock_counter ? "enabled" : "disabled");
```
a `rauc status` depends on `retrieve_status_via_dbus` - which isn't implemented yet
so the status would always say "disabled" - which is misleading
(and mis-lead me to debug into the code... sth we could save others from ;-)

Reply:
And of course, this should only be printed when counter locking is enabled!

Reply:
`r_status_formatter_readable` is called via `print_status` -> `status_start`
And there `print_status` is just called if `!ENABLE_SERVICE`
Or is there another way to call it?